src/ore/src/process.rs

```diff
@@ -42,7 +42,7 @@
 macro_rules! halt {
     ($($arg:expr),* $(,)?) => {{
         $crate::__private::tracing::warn!("halting process: {}", format!($($arg),*));
-        $crate::process::exit_thread_safe(2);
+        $crate::process::exit_thread_safe(3);
```

Comment:
```suggestion
        $crate::process::exit_thread_safe(166);
```

Can we use something in the 166-199 range?
Discussion https://materializeinc.slack.com/archives/CTESPM7FU/p1743687737423139?thread_ts=1743684376.799089&cid=CTESPM7FU